src/Speckle.Sdk/Credentials/AccountManager.cs

```diff
@@ -606,7 +606,7 @@ private static async Task<string> GetAccessCode(string server, string challenge,
       throw new Exception("Local auth flow failed to complete within the timeout window");
     }
 
-    if (task.IsFaulted)
+    if (task.IsFaulted && task.Exception is not null)
```

Comment:
Is there a chance where the task `IsFaulted` but the `task.Exception` is null?

Would we not want to log that?

Reply:
I don't think there's a practical chance of it.  An Exception is always thrown